### PR TITLE
Updated range-formats in "acceptable formats"

### PR DESCRIPTION
**Pull request by roberthstrand**
The examples showed in "Acceptable formats" used the word *through" instead of what is actually accepted in PowerShell. Updated them to reflect real world scenarios to avoid confusion.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks is needed for this community contributed PR. Thanks.

File: docset/windows/netsecurity/new-netfirewallrule.md [hunk 1]
```diff
@@ -416,7 +416,7 @@ Specifies the ICMP type codes.
 The key encoding is specified by running the Set-NetFirewallSetting cmdlet with the *KeyEncoding* parameter. 
 The acceptable values for this parameter are:
 
-- ICMP type code: 0 through 255. 
+- ICMP type code: 0-255.
```
Comment (JohanFreelancer9) [inline docset/windows/netsecurity/new-netfirewallrule.md]:
```suggestion
- ICMP type code: 0-255
```

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Looks good.

**Comment by dariomws306** [comment]
Looks good.

**Comment by e0i** [comment]
@get-itips Any additional thoughts? Thanks.

**Comment by get-itips** [comment]
Ok

**Comment by e0i** [comment]
@AndreaBarr This PR can be merged. Thanks.

**Pull request merged**